File: Work/WK/MS/MMService/MMService/Models/Post/CommentData.cs
namespace MMService.Models.Post
{
    /// <summary>
    /// 評論資料
    /// </summary>
    public class CommentData
    {
        /// <summary>
        /// 贴子 Id
        /// </summary>
        public string PostId { get; set; } = string.Empty;

        /// <summary>
        /// 消費時間
        /// </summary>
        public DateTime SpentTime { get; set; }

        /// <summary>
        /// 評論內容
        /// </summary>
        public string Comment { get; set; } = string.Empty;

        /// <summary>
        /// 地區代碼
        /// </summary>
        public string AreaCode { get; set; } = string.Empty;

        /// <summary>
        /// 照片 id list
        /// </summary>
        public string[]? PhotoIds { get; set; } = Array.Empty<string>();
    }
}

File: Work/WK/MS/Web_WebSV/Web/Web/Content/CTS/ClientApp/mimi-dating/src/models/CommentData.ts
export interface CommentData {
  /// 帖子 Id
  postId: string;

  /// 消費時間
  spentTime: string;

  /// 評論內容
  comment: string;

  /// 地區代碼
  areaCode: string;

  /// 照片 id list
  photoIds: string[];

  /// 照片對應 id/value
  photoSource: { [name: string]: string };
}
